docs/root/configuration/http/http_filters/connect_grpc_bridge_filter.rst

```diff
@@ -8,7 +8,7 @@ Connect-gRPC Bridge
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.connect_grpc_bridge.v3.FilterConfig>`
 
 This filter enables a Buf Connect client to connect to a compliant gRPC server.
-More information on the Buf Connect protocol can be found here https://connect.build/docs/protocol.
+More information on the Buf Connect protocol can be found `here <https://connect.build/docs/protocol>`.
```

Comment:
```suggestion
More information on the Buf Connect protocol can be found `here <https://connect.build/docs/protocol>`_.
```

Reply:
what does the `_` at the end do? (I am still building stuff locally to fix the warnings/errors)

Reply:
check "Exteranal hyperlink targets" here https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets

so ironic that you cant link properly to docs on the docutils site 8/

Reply:
aah, I can see it now in the preview. Fixed.